### PR TITLE
Add node-halt and node-reboot scripts

### PR DESCRIPTION
**Pull request by mabott**
Add node-halt and node-reboot scripts

### CODE REVIEW

File: node_halt.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+# the License.
+
+"""
+== Description:
+
+This script will reboot the node specified by the --host option
+
+== Script Usage:
+
+node_reboot.py --host ip_address|hostname [options]
```
Comment (Eric-Wright) [inline node_halt.py]: this should be node_halt.py

File: node_halt.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# Copyright (c) 2013 Qumulo, Inc.
```
Comment (Eric-Wright) [inline node_halt.py]: update copyright

File: node_halt.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+== Script Usage:
+
+node_reboot.py --host ip_address|hostname [options]
+
+
+=== Examples:
+
+- Run the script against a remote node
+SCRIPTNAME.py --host 192.168.194.333
```
Comment (Eric-Wright) [inline node_halt.py]: shouldn't these be node_halt.py instead of SCRIPTNAME.py?

File: node_halt.py [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""
+== Description:
+
+This script will reboot the node specified by the --host option
```
Comment (Eric-Wright) [inline node_halt.py]: s/reboot/halt/

File: node_reboot.py [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+== Script Usage:
+
+node_reboot.py --host ip_address|hostname [options]
+
+
+=== Examples:
+
+- Run the script against a remote node
+SCRIPTNAME.py --host 192.168.194.333
```
Comment (Eric-Wright) [inline node_reboot.py]: shouldn't these say node_reboot.py?

**Comment by mabott** [comment]
I was just adhering to the existing style in these scripts...

<_<

I will fix.

**Comment by mabott** [comment]
@Eric-Wright requested changes added to PR.

File: node_halt.py (hunk 4 above)
Comment (MichaelSEA) [inline node_halt.py]: see above.

File: node_halt.py [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        print "Node Halted\n"
+
+        return result
+
+
+def main():
+    """ main subroutine """
+
+    command = NodeHaltCommand()
+    command.node_halt()
```
Comment (MichaelSEA) [inline node_halt.py]: so: getting back from `halt` means “go to the fucking lab and recycle the cluster.  IE deleterious for people playing with this :wink:.  I’d put a big confirmation ‘Yes/No’ style like config_preserver on this cmdlet: https://github.com/Qumulo/config_preserver/blob/master/config_preserver.py#L227-L235

**Pull request merged**